.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
It would work for me with the following (not `root`, just plain peer trust)

```yml
defaults: &defaults
  adapter: postgresql
  host: <%= ENV.fetch('DB_HOST', 'localhost') %>
  username: <%= ENV.fetch('DB_USERNAME', 'null') %>
  password: <%= ENV.fetch('DB_PASSWORD', 'null') %>
  encoding: utf8
```